### ISSUE TITLE
Accessibility Audit: Radio buttons on eligibility questionnaire are not visible in high contrast mode

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#930]
In a Macbook, turn on Increase Contrast (System Preferences -> Accessibility -> Display -> 'Increase Contrast Mode'). Then go to your Firefox browser and go to Settings -> General -> Language & Appearance -> Colors, and then click the 'Only with High Contrast themes' option on 'Override the colors specified by the page with your selections above'. 

On a Windows machine, in Firefox, go to Settings -> Vision -> Use High Contrast and set 'Turn on high contrast' to On. 

Now go to Firefox and look at the [Age question](https://detroit-public-dev.netlify.app/eligibility/age) for the eligibility questionnaire. The radio button options don't appear.

NOTE: I was not able to replicate this with Chrome.

<img width="1177" alt="Screen Shot 2021-12-29 at 5 12 15 PM" src="https://user-images.githubusercontent.com/394908/147708087-d70f80b9-a022-49d3-96f7-2aaae69d3e65.png">
